prow/config.yaml

```diff
@@ -553,7 +553,7 @@ presubmits:
           value: /etc/ssh-key-secret/ssh-private
         - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
           value: /etc/ssh-key-secret/ssh-public
-        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170718-c3574ec6
+        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170720-69a2043d
```

Comment:
@wojtek-t Both of them are based off kubekins-e2e image. The image tag is made from a test-infra commit, https://github.com/kubernetes/test-infra/commit/69a2043d46ec84208edb2432ea7fc88ce10328de, which I bumped both e2e-prow and kubeadm base image tag. And when I push both image they are at the same commit.